server/game/gameSystems/MoveCardSystem.ts

```diff
@@ -3,20 +3,22 @@ import type { Card } from '../core/card/Card';
 import { CardType, EffectName, Location, WildcardCardType } from '../core/Constants';
 import * as EnumHelpers from '../core/utils/EnumHelpers';
 import { type ICardTargetSystemProperties, CardTargetSystem } from '../core/gameSystem/CardTargetSystem';
-import Contract from '../core/utils/Contract';
 
 export interface IMoveCardProperties extends ICardTargetSystemProperties {
     destination?: Location;
     switch?: boolean;
     switchTarget?: Card;
     shuffle?: boolean;
-    faceup?: boolean;
+    // TODO: remove completely if faceup logic is not needed
+    // faceup?: boolean;
     bottom?: boolean;
     changePlayer?: boolean;
     discardDestinationCards?: boolean;
 }
 
-/** @deprecated This system was imported from L5R but has not been tested */
+/**
+ * This system has been partially tested.
```

Comment:
I would call it about as tested as any other system at this point :P probably can remove this comment